impl/src/main/java/org/ehcache/loaderwriter/writebehind/BatchingLocalHeapWriteBehindQueue.java

```diff
@@ -200,7 +203,7 @@ public void run() {
             latest.remove(op.getKey(), op);
           }
         } finally {
-          LOGGER.info("Cancelling Batch " + this);
+          LOGGER.debug("Batch processing completed");
```

Comment:
The statement here is to inform that we are cancelling the scheduled batch expiry task, while it needed a level change and clarification, your change no longer reflects this.